src/main/java/io/mosip/mimoto/controller/IssuersController.java

```diff
@@ -59,18 +59,12 @@ public ResponseEntity<Object> getAllIssuers(@RequestParam(required = false) Stri
 
     @GetMapping("/{issuer-id}/wellknown")
```

Comment:
can we use ".well-known" like mentioned on spec instead of well-known ? since this API is going to give the complete well-known mock

Reply:
@abhip2565 let's make this change in mimoto as well as in Inji while consuming